resources/lib/context_animatedart.py

```diff
@@ -5,27 +5,45 @@
     Contextmenu for Animated art
 '''
 
+import os, sys
 import xbmc
 import xbmcgui
 from metadatautils import MetadataUtils
-from utils import log_msg
+if sys.version_info.major == 3:
+    from resources.lib.utils import log_msg
+else:
+    from utils import log_msg
```

Comment:
You shouldn't need this if else block - `from resources.lib.utils import log_msg` will work for both PY 2 and 3

Reply:
thx. I'll remove this check.